example/main.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/ocagent"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/trace"
+	"go.opencensus.io/zpages"
+)
+
+func main() {
+	zPagesMux := http.NewServeMux()
```

Comment:
Remember the comment suggestion from the website PR? Please apply that here too.

Reply:
SG, added comments here.